src/transformers/models/roberta/modeling_roberta.py

```diff
@@ -695,7 +702,7 @@ class RobertaModel(RobertaPreTrainedModel):
 
     """
 
-    # Copied from transformers.models.bert.modeling_bert.BertModel.__init__ with Bert->Roberta
+    # Adapted from transformers.models.bert.modeling_bert.BertModel.__init__
```

Comment:
My plan is to actually go through all the "adapted from" `forward()`s here and try to add SDPA support for as many of them as possible.

I already have a working draft to add SDPA support for Roberta as soon as this goes in (so it won't make sense to use Roberta as a reference model). The draft already covers Roberta, XLMRoberta, XLMRobertaXL, and Camembert (and reverts the copy-from for those).

Some of the other multimodal ones are a little trickier because the common SDPA tests don't work well with them, but I plan to give those a shot as well. The goal is to revert most of the "adapted from" back to "copied from" in the next few PRs.

Reply:
Awesome - it'll be great to have SDPA added for these models. In the meantime, whilst they still don't have support, I'd still like for the "copied from" headers to be updated. Happy for one different from roberta to be covered.

Reply:
Gotcha, I changed the "reference model" from Bert to ClapText. 

I had to remove the "Copied From" from ClapText, which would make that the new "copied-from source" for Bert-like models without SPDA support.


**Aside**: I did some thinking about the copied-from problem and this approach of choosing another reference model, and I feel like it has its downsides as well:

* If we make another change to Bert, how would we know to update ClapText as well so that the changes can be propagated out through fix-copies to other models?
* We're losing important information about the "original source/reference" of a model. For example, it would seem weird to someone reading the CamembertModel code to see that it's a derivative of ClapTextModel.

This seems like a bigger architectural discussion that's beyond the scope of this PR. If you know of an active discussion around how this can be improved, I'd love to join in on that conversation.